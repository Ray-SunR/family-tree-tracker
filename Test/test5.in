* A
* B
p
! A
! B
! C
* A
p
