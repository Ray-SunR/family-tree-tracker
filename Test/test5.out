C is parent of B
